### PR TITLE
editorconfig.yml: remove sarcastic tone from promotional error

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by mweinelt** [comment]
This kinda removes more than any potential sarcasm, but also a useful reference to find out more about editorconfig.

File: .github/workflows/editorconfig.yml [hunk 1]
```diff
@@ -38,4 +38,4 @@ jobs:
         cat "$HOME/changed_files" | nix-shell -p editorconfig-checker --run 'xargs -r editorconfig-checker -disable-indent-size'
     - if: ${{ failure() }}
       run: |
-        echo "::error :: Hey! It looks like your changes don't follow our editorconfig settings. Read https://editorconfig.org/#download to configure your editor so you never see this error again."
+        echo "::error :: Your changes do not follow editorconfig settings."
```
Comment (lelgenio) [inline .github/workflows/editorconfig.yml:41]:
```suggestion
        echo "::error :: Your changes do not follow editorconfig settings. Read https://editorconfig.org/#download to configure your editor."
```

**Review by lelgenio — CHANGES_REQUESTED** [review]

File: .github/workflows/editorconfig.yml (hunk 1 above)
Comment (ehmry) [reply .github/workflows/editorconfig.yml:41]: I don't need a tool to tell me where to put whitespace. **Fuck product placement.**

Comment (piegamesde) [reply .github/workflows/editorconfig.yml:41]: @ehmry please at least *try* to engage in a constructive and productive discussion.

-----

@lelgenio I'm not sure what's the benefit of directly linking to the downloads section. How about this instead:

```suggestion
        echo "::error :: Your changes do not follow editorconfig settings. See https://editorconfig.org/ for more information."
```

**Pull request closed without merging**